core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -111,6 +111,10 @@ public CommitFailedException wrapped() {
   }
 
   private static <T> Pair<List<T>, String> paginate(List<T> list, String pageToken, int pageSize) {
+    if (pageToken == null) {
+      pageToken = INITIAL_PAGE_TOKEN;
```

Comment:
Looks like java checkstyle doesn't like parameter reassignement,
```
> Task :iceberg-core:showDeprecationRulesOnRevApiFailure SKIPPED
Error: eckstyle] [ERROR] /home/runner/work/iceberg/iceberg/core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java:115:17: Assignment of parameter 'pageToken' is not allowed. [ParameterAssignment]
```

Let's go back to the previous implementation 
```
int start = (pageToken == null || INITIAL_PAGE_TOKEN.equals(pageToken)) ? 0 : Integer.parseInt(pageToken);
```